plugins/aep-edge-analytics-streaming-validation/index.ts

```diff
@@ -1,77 +0,0 @@
-/*
```

Comment:
is there a reason we're deleting the code for this plugin? Deleting the code won't actually remove the plugin

Reply:
We need to remove this plugin under "Edge Analytics" category.  It doesn't apply to Edge Analytics.

Reply:
Okay, but deleting the code will not remove the plugin from any category. We either need to find a more appropriate category or delete the plugin from the library if it's not needed.

Reply:
Revert this back, we will fix this in another PR.